google-cloud-video-live_stream/samples/acceptance/helper.rb

```diff
@@ -0,0 +1,52 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require "minitest/autorun"
+require "minitest/focus"
+require "minitest/rg"
+
+require "google/cloud/video/live_stream"
+
+require_relative "../../../.toys/.lib/sample_loader"
+
+class LiveStreamSnippetSpec < Minitest::Spec
+  let(:client) { Google::Cloud::Video::LiveStream.livestream_service }
+
+  let(:project_id) { ENV["GOOGLE_CLOUD_PROJECT"] || raise("missing GOOGLE_CLOUD_PROJECT") }
+
+  let(:location_id) { "us-central1" }
+  let(:input_id) { "my-input-test-#{(Time.now.to_f * 1000).to_i}" }
+  let(:input_name) { "projects/#{project_id}/locations/#{location_id}/inputs/#{input_id}" }
+  let(:input_location_path) { client.location_path project: project_id, location: location_id }
+
+  let :input do
+    operation = client.create_input(
+      parent: input_location_path,
+      input:  {
+        type: Google::Cloud::Video::LiveStream::V1::Input::Type::RTMP_PUSH
+      },
+      input_id: input_id
+    )
+    operation.wait_until_done!
+    input = operation.response
```

Comment:
@NivedhaSenthil This line fails the lint check but all of the tests pass. Inputs are long-running operations, and I'd like to set :input = to the response of the operation, which is the actual Live Stream input. What can be done here?

Reply:
sorry for the late response.. missed the ping here
we could just leave the response unassigned to a variable, just `operation.response` in the last line should set it to the let :input

Reply:
done